src/utils/shared/k8s/kubectl.go

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package k8s
+
+import "os/exec"
+
+func KubectlCmd(args ...string) *exec.Cmd {
+	args_ := []string{}
+	if *kubeconfig != "" {
+		args_ = append(args_, "--kubeconfig", *kubeconfig)
```

Comment:
Since only one instance of the `--kubeconfig` flag is allowed [^1]. I think it would be better to use the `KUBECONFIG` environment variable.

eg: ```
	cmd.Env = append(cmd.Environ(),fmt.Sprintf("KUBECONFIG=%s", *kubeconfig))
	```

[^1]: https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#merging-kubeconfig-files

Reply:
I often used the --kubeconfig and just got to konw the KUBECONFIG env.